timemachine/cpp/src/nonbonded_all_pairs.cu

```diff
@@ -335,7 +305,7 @@ void NonbondedAllPairs<RealType>::execute_device(
     kernel_idx |= d_du_dx ? 1 << 1 : 0;
     kernel_idx |= d_u ? 1 << 2 : 0;
 
-    kernel_ptrs_[kernel_idx]<<<ceil_divide(p_ixn_count_[0], tpb / warp_size), tpb, 0, stream>>>(
+    kernel_ptrs_[kernel_idx]<<<NONBONDED_KERNEL_BLOCKS, NONBONDED_KERNEL_THREADS_PER_BLOCK, 0, stream>>>(
```

Comment:
nit: should we use a standard case for constants? (`warp_size` vs. `NONBONDED_KERNEL_BLOCKS`)

Reply:
That makes sense to me. I prefer all caps, currently we have a mix.

Reply:
Going to open a separate PR that converts all of our non caps constants to caps